pymumble_py3/mumble.py

```diff
@@ -673,7 +675,14 @@ def treat_command(self, cmd):
             self.send_message(PYMUMBLE_MSG_TYPES_USERSTATE, userstate)
             cmd.response = True
             self.commands.answer(cmd)
-
+        elif cmd.cmd == PYMUMBLE_CMD_REMOVEUSER:
+            userremove = mumble_pb2.UserRemove()
+            userremove.session = cmd.parameters["session"]
+            userremove.reason = cmd.parameters["reason"]
+            userremove.ban = cmd.parameters["ban"]
```

Comment:
You don't use `Actor` field. Any reason ? You prefer to not have the bot name into the ban message ?
https://github.com/azlux/pymumble/blob/pymumble_py3/pymumble_py3/mumble_pb2.py#L732

Reply:
I don't know how, but protobufs auto fills the actor field to the bot's session. This can be proven by printing this during a kick callback caused by the bot: `print(bot.users.myself["session"], event.actor)`

Reply:
Ok, it's optional into the protocol, maybe it's to override the value.